cluster/charts/rook-ceph-cluster/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
If configuring the cluster in the same namespace as the operator we would need to skip the RBAC creation.